xcodeproj/internal/xcodeproj_incremental_rule.bzl

```
@@ -219,6 +219,9 @@ def _write_autogeneration_config_file(
     args = actions.args()
     args.set_param_file_format("multiline")
 
+    testOptions = config.get("test_options")
+    args.add(testOptions[0] if testOptions[0] != None else "")
+    args.add(testOptions[1] if testOptions[1] != None else "")
```

We make sure that these are non-empty before here so we can make this more efficient:
```suggestion
    args.add_all(config.get("test_options"))
```